apps/frontend/src/libs/components/input/styles.module.css

```
@@ -1,14 +1,75 @@
 .label {
 	display: flex;
 	flex-direction: column;
-	gap: 8px;
+	gap: 4px;
 	width: 100%;
-	color: var(--color-text-primary);
+	max-width: 364px;
+	font-family: var(--font-family);
 }
 
 .error {
-	font-size: 14px;
-	font-weight: 500;
-	line-height: 1.3;
-	color: var(--color-danger);
+	display: inline-flex;
+	gap: 7px;
```

gaps, paddings, and usual sizes should be divisible by 4 from our design logic. If it's not this way on Figma, it was a mistake 😅 
```suggestion
	gap: 8px;
```